index.js

```diff
@@ -650,3 +652,15 @@ export async function createKey({ json }) {
     console.log(key)
   }
 }
+
+export const reset = async () => {
+  const store = getStore()
+  const exportData = await store.load()
+  if (exportData) {
+    let data = AgentData.fromExport(exportData)
+    // do not reset the principal
+    data = await AgentData.create({ principal: data.principal, meta: data.meta })
```

Comment:
At first I read the code and didn't see how it would not just re-import the same proofs `fromExport`. This (imho) makes it more obvious what's going on. just a idea
```suggestion
    data = await AgentData.create({ principal: data.principal, meta: data.meta, delegations: undefined })
```